src/pages/iou/request/step/IOURequestStepScan/index.js

```diff
@@ -178,7 +188,31 @@ function IOURequestStepScan({
         }
 
         navigateToConfirmationStep();
-    }, [cameraRef, action, transactionID, updateScanAndNavigate, navigateToConfirmationStep]);
+    }, [action, transactionID, updateScanAndNavigate, navigateToConfirmationStep]);
+
+    const capturePhoto = useCallback(() => {
+        if (!cameraRef.current.getScreenshot) {
+            return;
+        }
+
+        if (trackRef.current && isFlashLightOn) {
+            trackRef.current
+                .applyConstraints({
+                    advanced: [{torch: true}],
+                })
+                .then(() => {
+                    setTimeout(() => {
+                        getScreenshot();
+                        trackRef.current.applyConstraints({
+                            advanced: [{torch: false}],
+                        });
+                    }, 2000);
```

Comment:
I think we need to clear the timeout on unmount, otherwise we'd be trying access undefined values